main/reference/zoe-api/zoe-contract-facet.md

```diff
@@ -235,7 +236,7 @@ zcf.assertUniqueKeyword(keyword);
 - Returns: None.
 
 Prohibit invocation of invitatations whose description include any of the strings.
-Any of the strings that end with a colon (:) will be treated as a prefix,
+Any of the strings that end with a colon (`:`) will be treated as a prefix,
 and invitations whose description string begins with the string (including the colon)
 will be burned and not processed if passed to **E(Zoe).offer()**.
```

Comment:
@Chris-Hibbert , are they actually burned? That surprises me.

Reply:
Your instincts are correct. If the invitation is blocked, we `Fail` out of the `offer()` call and don't get to the immediately following `burnInvitation()`.
```
    !instanceAdmin.isBlocked(description) ||
      Fail`not accepting offer with description ${q(description)}`;
    const { invitationHandle } = await burnInvitation(
      invitationIssuer,
      invitation,
    );
```